### PR TITLE
Name the kubeadm v1.7 jobs consistently

### PR DESCRIPTION
**Pull request by luxas**
Gah, noticed my inconsistency...
/assign @krzyzacy

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -1889,9 +1889,9 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-6
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
-  - name: ci-gce-kubeadm-1.7
+  - name: kubeadm-gce-1.7
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: actually `ci-` was kinda misleading here... maybe it should actually be postsubmit-kubeadm-gce-1.7?

Comment (krzyzacy) [inline testgrid/config/config.yaml]: I'm fine with this as well though.

**Comment by luxas** [comment]
Tests passed, let's merge this!

**Pull request merged**